app/upgrades/v20/upgrades.go

```diff
@@ -508,7 +508,6 @@ func MigrateConsumerRemovalProposal(
 
 	msgRemoveConsumer := providertypes.MsgRemoveConsumer{
 		ConsumerId: rmConsumerID,
-		StopTime:   msg.StopTime,
```

Comment:
@mpoke This means we don't have a "1-1" correspondence between `ConsumerRemovalProposal`s and `MsgRemoveConsumer`s because for example, there might be a `ConsumerRemovalProposal` to remove a chain in a year but with `MsgRemoveConsumer`, we'll stop it directly and remove it after an unbonding period of time.

Reply:
That's fine. There is no removal prop and I doubt there will be one.